### PR TITLE
Fixing User Preferences Design Issues

### PR DESCRIPTION
**Pull request by kma-odoo**
Task Link: https://www.odoo.com/web#id=2042276&action=327&model=project.task&view_type=form&menu_id=4720

Pad Link: https://pad.odoo.com/p/r.a91623a4f361ee1ad7681b73a0106cd8

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/scss/fields.scss [hunk 1]
```diff
@@ -363,6 +363,7 @@
             }
 
             > .fa {
+                padding: 4px;
```
Comment (qsm-odoo) [inline addons/web/static/src/scss/fields.scss]: The commit message explains something that this commit does not do.

Comment (kma-odoo) [reply addons/web/static/src/scss/fields.scss]: Done, changed the commit message.

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**